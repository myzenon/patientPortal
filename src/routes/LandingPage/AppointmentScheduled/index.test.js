import React from 'react'
import { AppointmentScheduled } from './index'
import ReactGA from 'react-ga';

   // This should be part of your setup
   ReactGA.initialize('UA-121783543-1', {
       'debug': true
     });;

jest.mock("react-ga")

test('renders correctly', () => {
    const props = {
        location:{"search":"none"},
        user: {
            ...mockObj.user
        },
        cmr: {
            ...mockObj.cmr
        },
        patient: {
            ...mockObj.patient,
            appointment: '2018-03-03T11:00:00-0800'
        },
        jestTest: true
    }
    const wrapper = render (
        <AppointmentScheduled {...props} />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
})


test('no appointment and cmr.complete = true', done => {
    const props = {
        location:{"search":"none"},
        user: {
            ...mockObj.user
        },
        cmr: {
            ...mockObj.cmr,
            complete: true
        },
        patient: {
            ...mockObj.patient,
            appointment: ''
        },
        push: (url) => {
            expect(url).toBe('/')
            done()
        }
    }
    render (
        <AppointmentScheduled {...props} />
    )
})
