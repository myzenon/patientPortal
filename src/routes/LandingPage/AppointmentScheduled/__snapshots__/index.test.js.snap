// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="content"
>
  <p
    class="titleName"
  >
    Carl, our pharmacist will call you on
  </p>
  <p
    class="scheduledDate"
  >
    March 3rd at 7:00pm UTC
  </p>
  <div
    class="text"
  >
    <p>
      <strong>
        What You Can Expect
      </strong>
    </p>
    <p>
      Our pharmacist will review all of your medications—including prescriptions, over-the-counter drugs, herbals and supplements—to help identify any drug-related problems you may be experiencing
    </p>
    <p>
      <strong>
        On the Call
      </strong>
    </p>
    <p>
      Our pharmacist will answer any questions you may have about your medications, and will also be looking for ways to reduce your prescription costs and improve your health. They will provide you with recommendations that are specific to you, and will follow up with your doctor as well.
    </p>
    <p>
      <strong>
        How do I Change My Appointment Time and Date?
      </strong>
    </p>
    <p>
      To change the time and date of your pharmacist call, just call us at 
      <strong />
      .
    </p>
    <p>
      <strong>
        Anything Else?
      </strong>
    </p>
    <p>
      Ask questions! Our pharmacists are here to serve you. There is no additional cost for this service, and the more we know about you, the more we can help!
    </p>
    <p>
      <strong>
        What Happens After the Appointment?
      </strong>
    </p>
    <p>
      After the call, our pharmacist will prepare two reports for you: a personalized medication list, and a medication action plan. These reports will be visible to you any time you log into the Arine portal.
    </p>
  </div>
  <div>
    <div
      class="wrapper"
    >
      <div
        class="stepCompleted"
      >
        Account
        <br />
        Setup
        <div
          class="stepNumberActive"
        >
          1
        </div>
      </div>
      <div
        class="stepCompleted"
      >
        Medication
        <br />
        Review
        <div
          class="stepNumberActive"
        >
          2
        </div>
      </div>
      <div
        class="stepCompleted"
      >
        Additional 
        <br />
        Medications
        <div
          class="stepNumberActive"
        >
          3
        </div>
      </div>
      <div
        class="stepCompleted"
      >
        Health
        <br />
        Updates
        <div
          class="stepNumberActive"
        >
          4
        </div>
      </div>
      <div
        class="stepActive"
      >
        Set
        <br />
        Appointment
        <div
          class="stepNumberActive"
        >
          5
        </div>
      </div>
    </div>
  </div>
</div>
`;
